### PR TITLE
Redis bot README.md file

### PR DESCRIPTION
**Pull request by pedromreis**
Just an explanation on the bot parameters and some usage examples.

### CODE REVIEW

File: intelmq/bots/outputs/redis/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* redis_server_ip   : remote server IP address, e.g.: 127.0.0.1
+* redis_server_port : remote server Port, e.g: 6379
+* redis_timeout     : Connection timout, in msecs, e.g.: 50000
+
+
+### Examples of usage:
+
+* Can be used to send events to be processed in another system. E.g.: send events to Logstash.
+
+* In a multitennent instalation can be used to send events to external/remote IntelMQ instance. Any expert bot queue can receive the events.
```
Comment (sebix) [inline intelmq/bots/outputs/redis/README.md]: Typo `instalation`.

**Comment by aaronkaplan** [comment]
Thx! will still fix the typo

**Comment by pedromreis** [comment]
Just fixed both typos... :)

**Comment by aaronkaplan** [comment]
okay :) you were faster hehe, thanks!! very nice work.

**Pull request merged**